front/lib/api/redis.ts

```diff
@@ -0,0 +1,40 @@
+import type { RedisClientType } from "redis";
+import { createClient } from "redis";
+
+import logger from "@app/logger/logger";
+
+let client: RedisClientType;
+
+export async function getRedisClient(): Promise<RedisClientType> {
+  if (!client) {
+    const { REDIS_URI } = process.env;
+    if (!REDIS_URI) {
+      throw new Error("REDIS_URI is not defined");
+    }
+
+    client = createClient({
+      url: REDIS_URI,
+      isolationPoolOptions: {
+        acquireTimeoutMillis: 10000, // 10 seconds.
+        // We support up to 200 concurrent connections for streaming.
+        max: 200,
+      },
+    });
+    client.on("error", (err) => logger.info("Redis Client Error", err));
```

Comment:
We need to enforce the logging format here:
```
logger.info({err}, "Redis Client Error");
```

Reply:
Done.